src/content/en/fundamentals/security/credential-management/save-forms.md

```diff
@@ -140,10 +137,14 @@ or proceed to the personalized page.
       e.preventDefault();
 
       // Try sign-in with AJAX
-      fetch(/'signin', {
+      let form = new FormData();
+      form.append('email', c.id);
+      form.append('password', c.password);
+
+      return fetch('/signin', {
         method: 'POST',
-        body: new FormData(e.target),
-        credentials: 'include'
+        credentials: 'include',
+        body: form
       }).then(res => {
```

Comment:
This is exact copy from https://developers.google.com/web/fundamentals/security/credential-management/save-forms and this should be fine because received object isn't a credential object:

```
// Get form's DOM object
var f = document.querySelector('#signup');
f.addEventListener('submit', e => {

  // Stop submitting form by itself
  e.preventDefault();

  // Try sign-in with AJAX
  fetch(/'signin', {
    method: 'POST',
    body: new FormData(e.target),
    credentials: 'include'
  }).then(res => {
```